SConstruct

```diff
@@ -109,6 +125,42 @@ def scons():
                   "create releases")
             exit(1)
 
+        variables = Variables()
+
+        variables.Add('RELEASE', 'Set to the release version to make', None)
+        variables.Add('RELEASE_BASE', 'Set to the release version to make',
+                      'master')
+        variables.Add('ORG_NAME', 'The GitHub project to do the release on.',
+                      'daos-stack')
+        variables.Add('REMOTE_NAME', 'The remoten name release on.', 'origin')
+
+        env = Environment(variables=variables)
+
+        org_name = env['ORG_NAME']
+        remote_name = env['REMOTE_NAME']
+        base_branch = env['RELEASE_BASE']
+
+        try:
+            tag = env['RELEASE']
```

Comment:
Does this actually throw an exception?  the default should be None

Reply:
It must:

```bash
$ scons release
scons: Reading SConscript files ...
Usage: scons RELEASE=x.y.z release
```

We can only get that error message if the exception is thrown.